packages/module/patternfly-docs/content/extensions/data-view/examples/Table/Table.md

```diff
@@ -0,0 +1,130 @@
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+import { useDataViewSelection, useDataViewSort } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { DataView, DataViewState } from '@patternfly/react-data-view/dist/dynamic/DataView';
+
+The **data view table** component renders your data into columns and rows within a [PatternFly table](/components/table) component. You can easily customize and configure the table by adjusting the properties listed at the bottom of this page.
+
+## Configuring rows and columns
+Below is a detailed description of properties used to define rows and columns for your table:
+- `columns`: Defines the column heads of the table. Each item in the array can be a `ReactNode` for simple heads, or an object with the following properties:
+  - `cell`: Content to display in the column head.
+  - `props` (optional): (`ThProps`) to pass to the `<Th>` component, such as `width`, `sort`, and other table head cell properties.
+- `rows`: Defines the rows to be displayed in the table. Each item in the array can be either an array of `DataViewTd` for simple rows, or an object with the following properties:
+  - `row`: Content to display in each cell in the row.
+  - `id` (optional): Unique identifier for the row that's used for matching selected items.
+  - `props` (optional): (`TrProps`) to pass to the `<Tr>` component, such as `isHoverable`, `isRowSelected`, and other table row properties.
+
+It is also possible to disable row selection using the `isSelectDisabled` function, which can be passed to the wrapping `DataView` component through the `selection`.
+
+### Table example
+```js file="./DataViewTableExample.tsx"
+
+```
+
+## Tree table
+
+A tree table includes expandable rows and custom icons for leaf and parent nodes. 
+To enable a tree table, pass the `isTreeTable` flag to the `<DataViewTable>` component.
+
+
+Tree table rows have to be defined with following keys:
+  - `row`: Defines the content for each cell in the row.
+  - `id`: Unique identifier for the row that's used for matching selected items.
+  - `children` (optional): Defines the children rows.
+
+To update a row's icon to reflect its expansion state, pass `collapsedIcon`, `expandedIcon`, and `leafIcon` to `<DataViewTable>`.
+
+To disable row selection, pass the `isSelectDisabled` function to `selection` prop of the wrapping `<DataView>` component .
+
+### Tree table example
+
+```js file="./DataViewTableTreeExample.tsx"
+
+```
+
+## Sorting
+The following example demonstrates how to enable sorting functionality within a data view. This implementation support dynamic sorting by column and persists the sort state in the page's URL via [React Router](https://reactrouter.com/).
+
+### Sorting example
+```js file="./SortingExample.tsx"
+
+```
+### useDataViewSort hook
```

Comment:
```suggestion
### Sorting state
```
reverting this from my suggestion to match the title we have in the similar toolbar example